docs/release-notes/release-notes-0-17-12.md

```diff
@@ -11,7 +11,7 @@ The following new features are included in this release.
 
 ### Function Workers
 
-Description of feature.
+DMYTRO - Please add a description of Function Workers here. It can be similar or the same as the intro to your documentation topic. Link the topic below.
```

Comment:
Heya @dlazarenko-macrometa! See the note in the doc here.

Reply:
@diana-macrometa  - I added the description. Please review it again